spec/02-integration/01-cmd/07-cluster_spec.lua

```diff
@@ -1,9 +1,55 @@
 local helpers = require "spec.helpers"
 
 describe("kong cluster", function()
-  it("keygen", function()
-    local _, stderr, stdout = helpers.kong_exec "cluster keygen"
+  before_each(function()
+    helpers.kill_all()
+  end)
+  teardown(function()
+    helpers.kill_all()
+    helpers.clean_prefix()
+  end)
+
+
+  it("cluster help", function()
+    local _, stderr = helpers.kong_exec "cluster --help"
+    assert.not_equal("", stderr)
+  end)
+  it("generates a key", function()
+    local _, stderr, stdout = assert(helpers.kong_exec("cluster keygen"))
     assert.equal("", stderr)
     assert.equal(26, stdout:len()) -- 24 + \r\n
   end)
+  it("shows members", function()
+    assert(helpers.kong_exec("start --conf "..helpers.test_conf_path))
+    local _, _, stdout = assert(helpers.kong_exec("cluster members --conf "..helpers.test_conf_path))
+    assert.matches("alive", stdout)
+  end)
+  it("shows rechability", function()
+    assert(helpers.kong_exec("start --conf "..helpers.test_conf_path))
+    local _, _, stdout = assert(helpers.kong_exec("cluster reachability --conf "..helpers.test_conf_path))
+    assert.matches("Successfully contacted all live nodes", stdout)
+  end)
+  it("force-leaves a node", function()
+    assert(helpers.kong_exec("start --conf "..helpers.test_conf_path))
+    local _, _, stdout = assert(helpers.kong_exec("cluster force-leave 127.0.0.1 --conf "..helpers.test_conf_path))
+    assert.matches("left node 127.0.0.1", stdout)
```

Comment:
Should be 
```lua
assert.matches("left node 127.0.0.1", stdout, nil, true)
```
Because arg #1 will be interpreted as a pattern otherwise, which means `.` can be any character. Better to have sane examples to not lead to errors in the future.